Comanche055/TVCSTROKETEST.agc

```diff
@@ -151,7 +151,7 @@ STRKCHK		INHINT
 		BZF	+3
 
 		CAE	ESTROKER	# BEGIN ON NEXT DAP PASS (PITCH OR YAW)
-		TS	STROKER		# (STROKING DONE IN PITCH ONLY, HOWEVER)
+		TS	STROKER		# (STROKING DONE IN PITCH ONLY. HOWEVER)
```

Comment:
❓  This looks to be one of the many badly rendered commas. I'll need to check the original scans for this one